### ISSUE TITLE
Many errors in Lab 2, Ex 3, Task 2

### ISSUE THREAD

**Issue by AnnetteGill** [MicrosoftLearning/MS-700-Managing-Microsoft-Teams#189]
# Module: 2
### Task: 2

Please could you review the instructions for task 2, because the formatting is awful and very hard to follow - experienced students were having trouble following the instructions.

1. There are several spelling mistakes, e.g. Lables instead of Labels.

2. The indentation of the instructions is wrong and therefore confusing.
For example, in step 4, a,b,c,d and e are indented. Then steps f, g and h are not indented. 
Then step 5 is indented. As are a, b and c.  But d is not indented. 
5e is made copiable - it shouldn't be, because it's just an instructio

3. Several sections of text are made copiable when they shouldn't be.
For example, as mentioned above, step 5e is copiable, but it shouldn't be - this is not text that the user should copy into their lab environment, it is simply an instruction.

4. After step 5e there are some bullet points, and then the next steps are marked with Romans numerals. Why?

5. In step 7iii, the checkboxes are now called "Content access" and "Apply content marking"

7. In step 7ii, "thislabel" should be "this label"

8. In step 5, "Select Infromation Protection" should be "Select Information Protection"

9. In step 5a, "baic" should be "basic"

**Comment by AnnetteGill** [comment]
Also, in step 4a, "seelct" should be "select"

**Comment by AnikaMicrosoft** [comment]
acknowledged. working on issues now.

**Comment by AnikaMicrosoft** [comment]
This should now be resolved.

**Comment by AnnetteGill** [comment]
Thank you.
